### ISSUE TITLE
[plugin-explorer] Shouldn't graphiql-explorer.d.ts be imported?

### ISSUE THREAD

**Issue by hasparus** [graphql/graphiql#2729]
Hey @thomasheyenbrock, I'm using `graphiql-plugin-explorer` (thanks for this amazing work!) and I noticed I'm missing types for `graphiql-explorer`. 

I added them with patch package to _types/index.d.ts_, but I'm wondering — won't including the types for `graphiql-explorer` be a problem if somebody publishes them to DefinitelyTyped? Do you think adding them to DT and setting a peer dep on `@types/graphiql-explorer` would be a safer solution?

---

_types/index.d.ts_
```diff
import { GraphiQLPlugin } from '@graphiql/react';
import { GraphiQLExplorerProps } from 'graphiql-explorer';
import './index.css';
+import './graphiql-explorer'
export declare function useExplorerPlugin(props: GraphiQLExplorerProps): GraphiQLPlugin;
```

**Comment by thomasheyenbrock** [comment]
Thanks for opening a PR for that already 👏

**Comment by thomasheyenbrock** [comment]
Just released the fix for this in `@graphiql/plugin-explorer@0.1.1`
